doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_ironside.rst

```diff
@@ -1012,6 +1012,17 @@ BOOTERROR
   * A non-zero value indicates that an error condition occurred, preventing the CPU from starting.
     Detailed information about the issue can be found in the boot report.
 
+.. _ug_nrf54h20_ironside_se_local_domain_reset:
+
+Local Domain Reset Handling
+****************************
+
+When a local domain experiences a reset |ISE| detects this event and triggers a global system reset to maintain system integrity.
+
+|ISE| uses the RESETHUB peripheral to monitor reset events across all local domains.
+When a local domain reset is detected, |ISE| responds by triggering a cold boot reset (``SYS_REBOOT_COLD``) to restart the entire system.
```

Comment:
This SYS_REBOOT_COLD doesn't map to any reset reason so I think it doesn't give a lot of info. Can't we instead say this triggers a global system reset and it will be marked as `SECSREQ` in RESETREAS.GLOBAL?